src/applications/rated-disabilities/util/datadog-rum/initializeRealUserMonitoring.js

```diff
@@ -0,0 +1,58 @@
+import { datadogRum } from '@datadog/browser-rum';
+
+import environment from '@department-of-veterans-affairs/platform-utilities/environment';
+
+const DISABLED_ENVIRONMENTS = ['production'];
```

Comment:
I only want to block usage in production currently. I'm not using a feature toggle currently because I want to see how far I can optimize page load times and having to wait for the feature toggle for this would slow things down